userguide/system.rst

```diff
@@ -2268,8 +2272,8 @@ licensed period, customer name, licensed features, and additional
 supported hardware.
 
 If the license expires or additional hardware, features, or
-contract type are required, contact your iXsystems support
-engineer. Once you have the new license string, click the
+contract type are required, contact an iXsystems support
+engineer. Once a new license string, click the
 :guilabel:`Update License` button, paste in the new license, and click
 :guilabel:`OK`. The new details will be displayed.
 
```

Comment:
s/Once a new license string, click the/If a new license string is provided, click the/